src/library/browse/browsetablemodel.cpp

```diff
@@ -160,10 +160,9 @@ TrackPointer BrowseTableModel::getTrack(const QModelIndex& index) const {
 TrackPointer BrowseTableModel::getTrackByRef(const TrackRef& trackRef) const {
     if (m_pRecordingManager->getRecordingLocation() == trackRef.getLocation()) {
         QMessageBox::critical(
-            0, tr("Mixxx Library"),
-            tr("Could not load the following file because"
-               " it is in use by Mixxx or another application.")
-            + "\n" + trackRef.getLocation());
+                0,
+                tr("Mixxx Library"),
+                tr("Could not load the following file because it is in use by Mixxx or another application.") + "\n" + trackRef.getLocation());
```

Comment:
Long line, requires Github vertical scrolling. Why does clang-format not kick in here?

Reply:
clang-format does kick in - if you look above it actually put all that stuff into one line in the first place. Maybe you need to tweak the penalty for long lines or smth like that.